src/rendering/batcher/shared/Batcher.ts

```diff
@@ -107,6 +111,8 @@ export interface BatcherOptions
 export class Batcher
 {
     public static defaultOptions: BatcherOptions = {
+        renderPipeId: 'batch',
+        floatsPerVertex: 6,
         vertexSize: 4,
         indexSize: 6,
```

Comment:
Perhaps it might be a good idea to rename those, since the name `vertexSize` is also used to refer to the "number of float32 per vertex", to make just there the naming doesn't cause any confusion.